<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Trajectory Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>0</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="energiesTab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>Adiab. Energies</string>
       </attribute>
       <layout class="QVBoxLayout" name="energiesLayout"/>
      </widget>
      <widget class="QWidget" name="coefficientsTab">
       <attribute name="title">
        <string>Coefficients</string>
       </attribute>
       <layout class="QVBoxLayout" name="coefficientsLayout"/>
      </widget>
      <widget class="QWidget" name="couplingsTab">
       <attribute name="title">
        <string>Nonadiab. Couplings</string>
       </attribute>
       <layout class="QVBoxLayout" name="couplingsLayout"/>
      </widget>
      <widget class="QWidget" name="currEnergyTab">
       <attribute name="title">
        <string>Curr. Energy</string>
       </attribute>
       <layout class="QVBoxLayout" name="currEnergyLayout"/>
      </widget>
      <widget class="QWidget" name="currStateTab">
       <attribute name="title">
        <string>Curr. State</string>
       </attribute>
       <layout class="QVBoxLayout" name="currStateLayout"/>
      </widget>
      <widget class="QWidget" name="populationsTab">
       <attribute name="title">
        <string>Populations</string>
       </attribute>
       <layout class="QVBoxLayout" name="populationsLayout"/>
      </widget>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="leftFrameLayout">
       <item row="1" column="0">
        <widget class="QListWidget" name="trajSelection">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="toolTip">
          <string>you can select multiple trajectories with Shift-Click of Cntrl-Click</string>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Trajectories:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="dirBrowser">
         <property name="toolTip">
          <string>enter path to directory with trajectories</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="openButton">
         <property name="toolTip">
          <string>open trajectory folder of top directory</string>
         </property>
         <property name="text">
          <string>Open...</string>
         </property>
         <property name="icon">
          <iconset theme="document-open">
           <normaloff/>
          </iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>802</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWidget" name="viewerWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>20</horstretch>
         <verstretch>20</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <layout class="QVBoxLayout" name="viewerWidgetLayout"/>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSlider" name="animationSlider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="animationTime">
          <property name="text">
           <string>Time: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="movieButton">
          <property name="text">
           <string>Movie...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOn_cluster">
   <property name="text">
    <string>on cluster</string>
   </property>
  </action>
  <action name="actionLocal">
   <property name="text">
    <string>local</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
